tests/browser-automated-tests-playwright/index.js

```diff
@@ -0,0 +1,213 @@
+require('dotenv').config()
+
+test_order_plan_with_subscription_and_upfront_charge = require('./tests/test_order_plan_with_subscription_and_upfront_charge');
+test_order_plan_with_only_upfront_charge = require('./tests/test_order_plan_with_only_upfront_charge');
+test_order_plan_with_only_recurring_charge = require('./tests/test_order_plan_with_only_recurring_charge');
+
+const playwright = require('playwright');
+const fs = require('fs');
+const devices = playwright.devices;
+const assert = require('assert');
+const DEFAULT_TIMEOUT = 10000;
+const PLAYWRIGHT_HOST = process.env.PLAYWRIGHT_HOST;
+const PLAYWRIGHT_HEADLESS = process.env.PLAYWRIGHT_HEADLESS.toLocaleLowerCase() == "true" || false;
+const videosDir = __dirname + '/videos/';
+const videoWidth = 1280
+const videoHeight = 720;
+const browserContextOptions = {recordVideo: { dir: videosDir, size: {width: videoWidth, height: videoHeight} }}
+
+//const browsers = ['chromium', 'webkit'];
+const browsers = ['chromium'];
+
+const iPhone = devices['iPhone 6'];
+
+// Delete any existing persons & subscriptions from the database
+async function clearDB() {
+  const browser = await playwright['chromium'].launch({headless: PLAYWRIGHT_HEADLESS});
+  const context = await browser.newContext(browserContextOptions);
+  const page = await context.newPage();
+
+  // Login then clearDB
+  await page.goto(PLAYWRIGHT_HOST + 'auth/login');
+  await page.fill('#email', 'admin@example.com');
+  await page.fill('#password', 'password');
+  await page.click('#login');
+
+  await page.goto(PLAYWRIGHT_HOST + '/admin/remove-subscriptions');
+  const contentSubscriptions = await page.evaluate(() => document.body.textContent.indexOf("all subscriptions deleted"));
+  assert(contentSubscriptions > -1);
+
+  await page.goto(PLAYWRIGHT_HOST + '/admin/remove-people');
+  const contentPeople = await page.evaluate(() => document.body.textContent.indexOf("all people deleted"));
+  assert(contentPeople > -1);
+
+  await page.goto(PLAYWRIGHT_HOST + '/admin/remove-transactions');
+  const contentTransactions = await page.evaluate(() => document.body.textContent.indexOf("all transactions deleted"));
+  assert(contentTransactions > -1);
+
+  await browser.close();
+}
+
+async function saveVideo(filename) {
+  const currentVideoFile = fs.readdirSync(videosDir).find(name => name.endsWith('webm'));
```

Comment:
Can't do this here, need to work out the most recent file (e.g. ls -ltr | head -n1) 
Or, maybe write a PR to Playwright to set video filename- at the moment playwright uses `createGuid` to randomly name the file (https://github.com/microsoft/playwright/blob/d40afa2fbf5100798386bd704797fe714d2e3e1b/src/server/webkit/wkPage.ts#L120) 

Ref https://github.com/microsoft/playwright/issues/4682

Other options 
- work out how to get the most recent file in a dir using nodejs